src/cpp/session/modules/SessionConnections.R

```diff
@@ -160,26 +160,107 @@ options(connectionObserver = list(
    NULL
 })
 
+.rs.addFunction("connectionReadSnippets", function() {
+   snippetsPath <- getOption("connections-path", "/etc/rstudio/connections/")
```

Comment:
In my view this option should be set on the server so that there's no way resetting R options inside a session could hide the connections supplied by an admin. What do you think? If we stick with an R option, we should probably prefix it with `rstudio.` for clarity.